src/test/java/com/github/iveina2021/page/AllProductsPage.java

```diff
@@ -0,0 +1,27 @@
+package com.github.iveina2021.page;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.page;
+
+// page_url = https://automationexercise.com/products
+
+public class AllProductsPage {
+    public SelenideElement allProductsLabel = $("h2[class$='text-center']");
```

Comment:
(weak selector) here use just search for header (h2) with centered text. It can be any header on the screen.

Reply:
Changed this selector